user/configure-device.rst

```diff
@@ -106,37 +72,46 @@ please add them):
         #option shared_secret ''
         #option consistent_key '1'
         #option mac_interface 'eth0'
+        #option management_interface 'tun0'
         #option merge_config '1'
         #option test_config '1'
         #option test_script '/usr/sbin/mytest'
-        #option uuid ''
-        #option key ''
-        list unmanaged 'system.@led'
-        list unmanaged 'network.loopback'
-        list unmanaged 'network.@switch'
-        list unmanaged 'network.@switch_vlan'
+        #option hardware_id_script '/usr/sbin/read_hw_id'
+        #option hardware_id_key '1'
+        option uuid ''
+        option key ''
```

Comment:
are you sure these 2 should be uncommented?

Reply:
This is the default template that you get right after installing openwisp-config. TBH I would rather remove this part because it provides no much value.

Reply:
Once you set the shared_secret and restart openwisp this file is totally overwritten

Reply:
I see now. It's fine for now. (uuid and key may still be entered manually in some cases so it's ok).